pyke/lightcurve.py

```diff
@@ -165,6 +167,37 @@ def remove_outliers(self, sigma=5.):
             new_lc.flux_err = self.flux_err[~outlier_mask]
         return new_lc
 
+    def bin(self, binsize=13):
+        """Bins a lightcurve.
+
+        Parameters
+        ----------
+        binsize : int
+            Size of the bin
+
+        Returns
+        -------
+        binned_lc : LightCurve object
+        """
+
+        binned_lc = copy.copy(self)
+        flux_err = None
+
+        if self.flux_err is not None:
+            n = len(self.flux)
+            flux_err_reshaped = self.flux_err[:-q or None].reshape(-1, binsize))
+            flux_err = math.sqrt(np.nansum((flux_err_reshaped[:-1] ** 2, axis=-1))) / n
```

Comment:
I'm just curious: why math.sqrt instead of np.sqrt?  Is it faster?

Reply:
should be `np.sqrt`, thanks!